pom.xml

```diff
@@ -58,7 +58,7 @@
     </parent>
 
     <properties>
-        <com.capitalone.dashboard.core.version>3.1.19</com.capitalone.dashboard.core.version>
+        <com.capitalone.dashboard.core.version>3.2.0-SNAPSHOT</com.capitalone.dashboard.core.version>
```

Comment:
Please use V3.2.0 of core and refrain from using snapshot versions

Reply:
done